tests/views/view_tests_base.py

```diff
@@ -10,28 +12,58 @@
 class ViewTestsBase(testutils.base.ServerTestsBase):
     """A base class for tests for the Django app."""
 
-    TEST_USER_EMAIL = 'kay@mib.gov'
-    TEST_USER_ID = 'k'
-
     def setUp(self):
         super(ViewTestsBase, self).setUp()
         self._xsrf_tool = utils.XsrfTool()
+        self.data_generator.admin_permission(
+            repo_id='global', email_address='z@mib.gov',
+            access_level=
+            admin_acls_model.AdminPermission.AccessLevel.SUPERADMIN,
+            expiration_date=datetime.datetime(2051, 1, 20))
```

Comment:
does that mean that 32 years from now, tests will start failing? :^p

Reply:
Probably the `datetime` library will be deprecated or changed in a non-backwards-compatible fashion before then, so someone will have to change this anyway :)